modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/folders/SuggestedFolderPodcastsPage.kt

```diff
@@ -0,0 +1,125 @@
+import au.com.shiftyjelly.pocketcasts.compose.Devices
+import au.com.shiftyjelly.pocketcasts.compose.components.PodcastImage
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP40
+import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.images.R
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+fun SuggestedFolderPodcastsPage(
+    folder: SuggestedFolder?,
+    onGoBackClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val folderColor = folder?.colorIndex
+        ?.let { MaterialTheme.theme.colors.getFolderColor(it) }
+        ?: MaterialTheme.theme.colors.primaryInteractive01
+
+    Column(
+        modifier = modifier,
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier.heightIn(min = 64.dp),
+        ) {
+            IconButton(
+                onClick = onGoBackClick,
+            ) {
+                Icon(
+                    painter = painterResource(R.drawable.ic_arrow_back),
+                    contentDescription = stringResource(LR.string.back),
+                    tint = folderColor,
+                )
+            }
+
+            TextH30(
+                text = folder?.name.orEmpty(),
+                color = folderColor,
+                maxLines = 1,
+                textAlign = TextAlign.Center,
+                modifier = Modifier.weight(1f),
+            )
+
+            Spacer(
+                modifier = Modifier.size(48.dp),
+            )
+        }
+
+        Spacer(
+            modifier = Modifier.height(4.dp),
+        )
+
+        if (folder != null) {
+            LazyVerticalGrid(
+                columns = GridCells.Adaptive(110.dp),
+                horizontalArrangement = Arrangement.spacedBy(6.dp),
+                verticalArrangement = Arrangement.spacedBy(6.dp),
+                modifier = Modifier
+                    .weight(1f)
+                    .padding(bottom = 8.dp)
+                    .padding(horizontal = 16.dp),
+            ) {
+                items(folder.podcastIds) { podcastId ->
+                    PodcastImage(podcastId)
+                }
+            }
+        } else {
+            Box(
+                contentAlignment = Alignment.Center,
+                modifier = Modifier.fillMaxSize(),
+            ) {
+                TextP40(
+                    text = stringResource(LR.string.error_generic_message),
+                )
+            }
+        }
+    }
+}
+
+@Preview(device = Devices.PortraitRegular)
+@Composable
+private fun SuggestedFolderPodcastsPagePreview(
+    @PreviewParameter(ThemePreviewParameterProvider::class) themeType: Theme.ThemeType,
+) {
+    AppThemeWithBackground(Theme.ThemeType.LIGHT) {
```

Comment:
Changing preview to use `themeType`:
```suggestion
    @PreviewParameter(ThemePreviewParameterProvider::class) themeType: Theme.ThemeType,
) {
    AppThemeWithBackground(themeType) {
```